pages/map.py

```diff
@@ -268,9 +348,17 @@ def store_trips_map_data(trips_data):
             color = f'hsl({ind * k}, 100%, 50%)'
             trips = trips_group_by_user_id.get_group(user_id).sort_values('trip_start_time_str').to_dict("records")
             users_data[user_id] = {'color': color, 'trips': trips}
-            for trip in trips:
-                coordinates['lon'].append(trip['start_coordinates'][0])
-                coordinates['lon'].append(trip['end_coordinates'][0])
-                coordinates['lat'].append(trip['start_coordinates'][1])
-                coordinates['lat'].append(trip['end_coordinates'][1])
-    return {'users_data': users_data, 'coordinates': coordinates}
+    groupped_data = []
+    groupped_data.append({'users_data':users_data})
```

Comment:
why are you combining the users_data from the user_id groups with the one from the modes?
Is `grouped_data` going to have two separate entries called `users_data`?
Do you need both? How do you know which is which?
If you do need both, why not make it clearer how to distinguish between them?